code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -24,9 +32,13 @@
 	var/mob/living/carbon/human/H = M
 	if(istype(H))
 		user.visible_message("<span class='warning'>\The [user] is trying to inject \the [M] with \the [src]!</span>","<span class='notice'>You are trying to inject \the [M] with \the [src].</span>")
+		var/inj_time = time
+		if(fast)

```

Comment:
hoookay. this doesn't make sense. Why use two variables, a boolean and a number, when you can just have a number that you can set to 0 when needed...?

Reply:
I, uh. It seemed like a good idea at the time?